designer/server/src/views/forms/partials/overview-draft.njk

```diff
@@ -0,0 +1,59 @@
+{% from "govuk/components/tag/macro.njk" import govukTag %}
```

Comment:
Partials are good, thanks for adding support for them

Not a blocker, but these two partials (draft, live) lend themselves to being components + tests?

E.g. Make it once but tell it which "state" to render in:

```njk
{% if form.draft %}
  {{ appFormInformation({
    metadata: form,
    state: 'draft'
  }) }}
{% endif %}
```

If you need conditional array items, you've got `.push()` available in Nunjucks:

```njk
{% set items = [
  {
    {
      key: {
        text: "Example 1"
      },
      value: {
        html: "Value 1"
      }
    }
  }
] %}

{% if params.metadata.xxx %}
  {% set items = (items.push([
    key: {
      text: "Example 2"
    },
    value: {
      html: "Value 2"
    }
  ]), items) %}
{% endif %}
```

Can always consolidate the two of them at a later date

Reply:
Yes I have used this `push` njk trick before. It's ugly but it works I guess.

As soon as we need to reuse these in a different context, I'll consider a macro :+1: